### DISCUSSION TITLE
Make the Mod Menu Screen Better

### DISCUSSION THREAD

**Discussion by sillyangel** [eaglerforge/EaglerForge#29] [Ideas]
just make the Mod menu screen look better, and have config options like how is inside of Mod Menu on java looks like

![wp83YOP](https://github.com/eaglerforge/EaglerForge/assets/50461810/49b84719-0915-4b67-a4be-b4306327dcd0)

**Comment by stpv221** [comment]
similar to the "meta" for mods post.

**Comment by stpv221** [comment]
Ye i think that it should be done along with the IndexedDB

**Comment by leahanderson1** [comment]
we're working on it

**Comment by ZXMushroom63** [comment]
here's an idea:
You could store mod metadata in userscript format.
For example:
```javascript
// @name=My New Mod
// @description=Description
// @version=1.0.0
// @icon=data:image/png;base64,......
```
This makes adding metadata to already existing mods easy without any breaking changes.

**Reply by leahanderson1** [reply]
i think this would be great

**Reply by STPv22** [reply]
this is better than my oruginal idea where mods are stored in a resource pack-like format

**Comment by leahanderson1** [comment]
new java gui added in [33bf7bb](https://github.com/eaglerforge/EaglerForge/commit/33bf7bbfadfdc47e98fd26417d5a038851aee664)
